ports.mk

```diff
@@ -18,6 +18,8 @@ install:
 	$(INSTALL) -c -m 644 $(DIST_FILES_DOCS) "$(DESTDIR)$(docdir)"
 	$(INSTALL) -d "$(DESTDIR)$(datadir)"
 	$(INSTALL) -c -m 644 $(DIST_FILES_THEMES) $(DIST_FILES_ENGINEDATA) "$(DESTDIR)$(datadir)/"
+	$(INSTALL) -d "$(DESTDIR)$(datadir)/applications"
```

Comment:
You have weird definition of `datadir`. It expands to `/usr/share/scummvm/` instead of `usr/share/`.